### ISSUE TITLE
warning on build

### ISSUE THREAD

**Issue by jwhitcraft** [bedrock-viz/bedrock-viz#42]
First off let me say, great project, when I map out my world, i get the following warnings

```
[warning] Unknown uname: minecraft:smooth_stone
[warning] Unknown uname: minecraft:stone_stairs
```

**Comment by jasper-wan** [comment]
that is okay, these warnings will not affect the map

we will add more missing unames in next version

**Comment by cabbey** [comment]
Fixed in v0.1.3
